ddtrace/contrib/sqlalchemy/engine.py

```diff
@@ -32,6 +34,20 @@ def trace_engine(engine, tracer=None, service=None):
     EngineTracer(tracer, service, engine)
 
 
+def _wrap_create_engine(func, module, args, kwargs):
+    """Trace the SQLAlchemy engine, creating an `EngineTracer`
+    object that will listen to SQLAlchemy events. A PIN object
+    is attached to the engine instance so that it can be
+    used later.
+    """
+    # the service name is set to `None` so that the engine
+    # name is used by default; users can update this setting
+    # using the PIN object
+    engine = func(*args, **kwargs)
+    EngineTracer(ddtrace.tracer, None, engine)
```

Comment:
Here we can change the `EngineTracer` signature so that `service=None` kwarg is available. I think that we can safely remove entirely the concept of `EngineTracer`, but removing it and/or adding a kwarg to the signature may introduce a breaking change. I prefer to keep everything as is, and adds the `patch()` method on top of what we have already.